src/libs/intents/localMethods.ts

```diff
@@ -46,25 +49,22 @@ const backToHome = (): Promise<null> => {
 /**
  * Get the fetchSessionCode function to be called with current CozyClient instance
  * fetchSessionCode gets a session code from the current cozy-client instance
- *
- * @param {CozyClient} client - current CozyClient instance
- *
- * @returns {Function|null}
- * @throws error containing invalid session code result
  */
 const fetchSessionCodeWithClient = (
   client?: CozyClient
-): (() => Promise<null>) => {
+): (() => Promise<string | null>) => {
   return async function fetchSessionCode() {
     if (!client) {
       return null
     }
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
-    const sessionCodeResult = await client.getStackClient().fetchSessionCode()
 
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    const sessionCodeResult = await (
+      client.getStackClient() as {
```

Comment:
Maybe we should declare those types somewhere else. I think that doing this inline makes the code less readable (too much noise).

Reply:
I agree, I will start adding these types into cozy-client directly so we can remove them. In the meantime I improved readability by removing inline typing